labs-ios-starter/Profiles/Model Controllers/ProfileController.swift

```diff
@@ -12,15 +12,24 @@ import OktaAuth
 class ProfileController {
     
     static let shared = ProfileController()
-    
-    lazy var oktaAuth = OktaAuth(baseURL: baseURL,
+
+    /// Apollo API base URL
+    let baseURL = URL(string: "https://apollo-a-api.herokuapp.com")!
+
+    ///Unit Testing URL
+    private let testURL = URL(string: "https://kd-apollo.herokuapp.com/")!
+
+    ///Okta auth URL
+    private let oktaAuthURL = URL(string: "https://dev-625244.okta.com/")!
+
+    lazy var oktaAuth = OktaAuth(baseURL: oktaAuthURL,
```

Comment:
fix indentation of parameters

Reply:
if you're referring to the oktaAuth parameter's indentation, it's the way it's meant to be - that's the convention in Swift. You may be thinking of Python where parameters are lined up by the trailing semicolon rather than the first letter of the parameter